### PR TITLE
Fix ui test timeouts

### PR DESCRIPTION
**Pull request by jrbenny35**
This should fix the "Timed Out" failures on circleci.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ workflows:
   scheduled-workflow:
     triggers:
       - schedule:
-          cron: "0 0,2 * * *"
+          cron: "0 0,12,15,18,21 * * *"
```
Comment (bobsilverberg) [inline .circleci/config.yml]: Just wondering, why this change?

Comment (jrbenny35) [reply .circleci/config.yml]: The integration tests job runs at certain times. This should allow it to run more throughout the development day instead of just at 0:00 UTC time.
